src/components/screens/SendScreen/screens/TransactionProcessingScreen.tsx

```diff
@@ -0,0 +1,330 @@
+            <Text md medium secondary>
+              Fee
+            </Text>
+          </View>
+          <Text md medium>
+            0.00001 XLM
+          </Text>
+        </View>
+      </View>
+
+      {/* View on stellar.expert button */}
+      <Button
+        variant="tertiary"
+        size="lg"
+        onPress={handleViewOnExplorer}
+        icon={
+          <Icon.LinkExternal01
+            size={16}
+            color={themeColors.foreground.primary}
+          />
+        }
+        iconPosition={IconPosition.RIGHT}
+      >
+        View on stellar.expert
+      </Button>
+    </View>
+  );
+};
+
+type TransactionProcessingScreenProps = {
+  selectedBalance: PricedBalance | undefined;
+  tokenValue: string;
+  address: string;
+  onClose: () => void;
+};
+
+const TransactionProcessingScreen: React.FC<
+  TransactionProcessingScreenProps
+> = ({ selectedBalance, tokenValue, address, onClose }) => {
+  const { t } = useAppTranslation();
+  const { themeColors } = useColors();
+  const navigation = useNavigation();
+  const slicedAddress = `${address.slice(0, 6)}...${address.slice(-4)}`;
+  const [isCompleted, setIsCompleted] = useState(false);
+  const bottomSheetModalRef = useRef<BottomSheetModal>(null);
+
+  // Hide the header when this component mounts
+  useEffect(() => {
+    navigation.setOptions({
+      headerShown: false,
+    });
+  }, [navigation]);
+
+  // Simulate transaction completion
+  // This will be replaced with actual transaction logic in the future
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setIsCompleted(true);
+    }, 2000);
+
+    return () => clearTimeout(timer);
+  }, []);
+
+  const handleClose = () => {
+    // First call the original onClose function
+    onClose();
+
+    // Reset the navigation to the main tab and set the initial route to History
+    navigation.dispatch(
+      CommonActions.reset({
+        index: 0,
+        routes: [
+          {
+            name: ROOT_NAVIGATOR_ROUTES.MAIN_TAB_STACK,
+            state: {
+              index: 0,
+              routes: [{ name: MAIN_TAB_ROUTES.TAB_HISTORY }],
+            },
+          },
+        ],
+      }),
+    );
+  };
+
+  const handleViewTransaction = () => {
+    // Show transaction details in bottom sheet
+    bottomSheetModalRef.current?.present();
+  };
+
+  return (
+    <BaseLayout insets={{ top: false }}>
+      <View className="flex-1 justify-between">
+        {/* Middle content */}
+        <View className="flex-1 items-center justify-center">
+          <View className="items-center gap-[8px]">
+            {isCompleted ? (
+              <Icon.CheckCircle size={48} color={themeColors.status.success} />
+            ) : (
+              <Spinner size="large" color={themeColors.base[1]} />
+            )}
+
+            <Display xs medium>
+              {isCompleted
+                ? t("transactionProcessingScreen.sent", "Sent!")
```

Comment:
is this correct? I'm not finding any `transactionProcessingScreen.sent` on the translation files